cibuildwheel/options.py

```diff
@@ -408,6 +408,7 @@ def build_options(self, identifier: Optional[str]) -> BuildOptions:
             environment_config = self.reader.get(
                 "environment", table={"item": '{k}="{v}"', "sep": " "}
             )
+            passenv = self.reader.get("passenv", sep=",").split(",")
```

Comment:
Some users might naturally write `CIBW_PASSENV="VAR_A VAR_B"` since the delimiter in the CIBW_ENVIRONMENT option is a space. Perhaps we should change the delimiter to a space, to match it?

Reply:
My thought was to match extras, but I considered space too; that would be fine. To me it's more of a list, while ENVIRONMENT is a dict, but I see why someone might compare them more than EXTRA (which really is a list).

Reply:
I've implemented it with space this time.